spec/requests/partners_requests_spec.rb

```diff
@@ -271,10 +271,25 @@
   end
 
   describe "POST #invite" do
+    subject { post invite_partner_path(default_params.merge(id: partner)) }
+    
+    let(:partner) do
+      partner = create(:partner, :uninvited, organization: @organization)
+      partner.profile.primary_user.delete
+      partner.profile.reload
+      partner
+    end
+
     it "send the invite" do
-      post invite_partner_path(default_params.merge(id: create(:partner, organization: @organization)))
+      subject
       expect(response).to have_http_status(:found)
     end
+
+    it 'sends email' do
+      subject
+      expect(ActionMailer::Base.deliveries.last.subject).to eq("You've been invited to be a partner with #{@organization.name}")
+      expect(ActionMailer::Base.deliveries.last.html_part.body).to include("You've been invited to become a partner organization with <strong>#{@organization.name}!</strong>")
```

Comment:
Hi @cielf , this test check subject and body, what do you think?

Reply:
@italomatos I think we don't need to test the text in both the mailer spec  and the requests spec -- following the patterns that I see in the current tests, I would use the mailer spec for testing the text.      Does that make sense to you?